test/unit/widgets/error.test.ts

```diff
@@ -5,11 +5,17 @@ import { errorMessageBlock } from '../../../src/widgets/error'
 jest.mock('solid-auth-client')
 const dom = new JSDOM('<!DOCTYPE html><p>Hello world</p>').window.document
 
-describe('button', () => {
+describe('errorMessageBlock', () => {
   it('exists', () => {
     expect(errorMessageBlock).toBeInstanceOf(Function)
   })
-  it('runs', () => {
-    expect(errorMessageBlock(dom, '', undefined)).toBeTruthy()
+  it('creates an error message div', () => {
+    expect(errorMessageBlock(dom, 'my error message', undefined)).toMatchSnapshot()
+  })
+  it('escapes the HTML', () => {
+    expect(errorMessageBlock(dom, 'my <script> kiddie </script> message', undefined)).toMatchSnapshot()
```

Comment:
Might want to check this more explicitly (e.g. using toMatch and making sure that < and > is not present in text), since a snapshot doesn't really test this in a good way I think.